hydra/core/override_parser/types.py

```diff
@@ -235,6 +236,16 @@ class Override:
     # Configs repo
     config_loader: Optional[ConfigLoader] = None
 
+    def __post_init__(self) -> None:
+        if self.pkg2 is not None:
+            # DEPRECATED: remove in 1.1
+            msg = (
+                "\nSupport for renaming packages via the command line "
+                "is deprecated since Hydra 1.0.5 and will be removed in Hydra 1.1. "
+                "For more details, see https://github.com/facebookresearch/hydra/issues/1140."
+            )
```

Comment:
This is one super long error message.
some newlines?
Also, format it with dedent(f""""""). see other places in the codebase.